appinfo/app.php

```diff
@@ -78,6 +78,8 @@ function () use ($c, $appName) {
 		Util::addScript($appName, 'slideshowcontrols');
 		Util::addScript($appName, 'slideshowzoomablepreview');
 		Util::addScript($appName, 'gallerybutton');
+		Util::addScript($appName, 'rotate');
+		Util::addScript($appName, 'rotacion_cup');
```

Comment:
I guess this name ``rotacion`` is the Spanish for the English ``rotation``. It makes no difference to the app working - all uses of ``rotacion_cup`` in this PR are consistent.